metabrainz/templates/reports/financial_reports/index.html

```diff
@@ -1,365 +1,365 @@
-    <li>September: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150930.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150930.pdf') }}">profit &amp; loss</a></li>
-    <li>August: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150831.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150831.pdf') }}">profit &amp; loss</a></li>
-    <li>July: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150731.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150731.pdf') }}">profit &amp; loss</a></li>
-    <li>June: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150630.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150630.pdf') }}">profit &amp; loss</a></li>
-    <li>May: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150531.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150531.pdf') }}">profit &amp; loss</a></li>
-    <li>April: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150430.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150430.pdf') }}">profit &amp; loss</a></li>
-    <li>March: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150331.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150331.pdf') }}">profit &amp; loss</a></li>
-    <li>February: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150228.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150228.pdf') }}">profit &amp; loss</a></li>
-    <li>January: <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_balance_sheet_20150131.pdf') }}">balance sheet</a>, <a href="{{ url_for('financial_reports.static', filename='MetaBrainz_Foundation_profit_loss_20150131.pdf') }}">profit &amp; loss</a></li>
+    <li>{{ _('December: <a href="{{ url_for(\'financial_reports.static\', filename=\'MetaBrainz_Foundation_balance_sheet_20151231.pdf\') }}">balance sheet</a>, <a href="{{ url_for(\'financial_reports.static\', filename=\'MetaBrainz_Foundation_profit_loss_20151231.pdf\') }}">profit &amp; loss') }}</a></li>
```

Comment:
I'd suggest not to wrap whole strings here, but months and other parts separately instead.